client/app/components/item-detail-page/template.hbs

```diff
@@ -30,55 +32,93 @@
+                    {{else}}
+                        <div class="loader text-center">
+                            <div class="loader-inner ball-pulse ball-pulse-dark">
+                              <div></div>
+                              <div></div>
+                              <div></div>
+                            </div>
+                        </div>
+                    {{/if}}
                 {{/if}}
             </div>
-        <div class="{{if (or item.hasFile item.hasWiki) 'col-sm-4' 'col-sm-12'}}">
+        <div class="{{if (or constructedItem.viewContent.file.visible constructedItem.viewContent.wiki.visible) 'col-sm-4' 'col-sm-12'}}">
             <h4 class="p-v-md f-w-md"><strong>Description</strong></h4>
-            {{#if item.node.description}}
-                <p>{{item.node.description}}</p>
-            {{else}}
-                <div class="loader text-center">
-                    <div class="loader-inner ball-pulse ball-pulse-dark">
-                      <div></div>
-                      <div></div>
-                      <div></div>
+            {{#if constructedItem.viewContent.description.visible}}
+                {{#if constructedItem.viewContent.description.loaded}}
+                    <p>{{constructedItem.viewContent.description.value}}</p>
+                {{else}}
+                    <div class="loader text-center">
+                        <div class="loader-inner ball-pulse ball-pulse-dark">
+                          <div></div>
+                          <div></div>
+                          <div></div>
+                        </div>
                     </div>
-                </div>
+                {{/if}}
             {{/if}}
 
             <h4 class="p-v-md f-w-md"><strong>Status</strong></h4>
             <p class="coll-item-status {{item.status}}"> {{item.status}} </p>
-            <h4 class="p-v-md f-w-md"><strong>Tags</strong></h4>
-            {{#if item.node.tags}}
-                <div>
-                    {{#each item.node.tags as |tag|}}
-                        <span class="coll-item-tag">{{tag}}</span>
-                    {{/each}}
-                </div>
-            {{else}}
-                <div class="loader text-center">
-                    <div class="loader-inner ball-pulse ball-pulse-dark">
-                      <div></div>
-                      <div></div>
-                      <div></div>
+
+            {{#if constructedItem.viewContent.tags.visible}}
+                <h4 class="p-v-md f-w-md"><strong>Tags</strong></h4>
+                {{#if constructedItem.viewContent.tags.loaded}}
+                    <div>
+                        {{#each item.node.tags as |tag|}}
+                            <span class="coll-item-tag">{{tag}}</span>
+                        {{/each}}
+                    </div>
+                {{else}}
+                    <div class="loader text-center">
+                        <div class="loader-inner ball-pulse ball-pulse-dark">
```

Comment:
This spinner never goes away if a project does not have tags. I believe this was an issue before, so feel free to fix it in a separate PR